### PR TITLE
Azure quickstart template for AS2 Send Receive scenarios

### PR DESCRIPTION
**Pull request by vinaysingh123**
Added a new quickstart template for AS2 Send Receive scenario using Azure Logic Apps.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@vinaysingh123__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (vinaysin). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by vinaysingh123** [comment]
@gatneil , @singhkays , @bmoore-msft - Can you please review it?

File: 201-logic-app-as2-send-receive/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+![Image of Contoso-AS2Receive run history](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-logic-app-as2-send-receive/images/contoso-as2receive-runhistory.png "Contoso-AS2Receive run history"))
+- The FabrikamFinance-AS2Send and Contoso-Receive logic apps demonstrate async send receive scenario. Open the FabrikamFinance-AS2Send Logic App blade and click on Run Trigger to run it.
+-- The async MDN is received by the FabrikamFinance-AS2ReceiveMDN logic app.
+![Image of FabrikamFinance-AS2ReceiveMDN run history](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-logic-app-as2-send-receive/images/fabrikamfinance-as2receivemdn-runhistory.png "FabrikamFinance-AS2ReceiveMDN run history"))
+-- Again, you can look into the run history and input/output for each action for these logic apps.
+
+## Notes
+
+Learn more about: Azure Logic Apps
+* **Azure Logic Apps** - https://azure.microsoft.com/en-us/documentation/articles/app-service-logic-what-are-logic-apps/
```
Comment (gatneil) [inline 201-logic-app-as2-send-receive/README.md]: remove en-us. Leaving this culture code out ensures that the viewer gets docs in their own language

File: 201-logic-app-as2-send-receive/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+* **Logic Apps Enterprise Integration Pack** - https://blogs.msdn.microsoft.com/logicapps/2016/06/30/public-preview-of-logic-apps-enteprise-integration-pack/
+* **B2B Processing capabilities in Logic Apps** - https://azure.microsoft.com/en-us/documentation/articles/app-service-logic-enterprise-integration-b2b/
```
Comment (gatneil) [inline 201-logic-app-as2-send-receive/README.md]: same here

File: 201-logic-app-as2-send-receive/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,1279 @@
...
+            }
+          },
+          "contentVersion": "1.0.0.0",
+          "outputs": { }
+        },
+        "parameters": {
+          "$connections": {
+            "value": {
+              "as2": {
+                "id": "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Web/locations/', parameters('logicAppLocation'), '/managedApis/', 'as2')]",
```
Comment (gatneil) [inline 201-logic-app-as2-send-receive/azuredeploy.json]: I'm used to seeing ids of the form /subscriptions/sub-id/resourceGroups/rg-id/providers/...

Does /subscriptions/sub-id/providers/... work as well? If so, then this is fine. If not, then you should probably use the resourceId function to simplify the id string

File: 201-logic-app-as2-send-receive/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,1279 @@
...
+                "connectionId": "[concat(resourceGroup().id, '/providers/Microsoft.Web/connections/', parameters('fabrikam_AS2_Connection_Name'))]",
```
Comment (gatneil) [inline 201-logic-app-as2-send-receive/azuredeploy.json]: same here; I would expect there to be some sort of /subscriptions/sub-id/ string at the beginning

File: 201-logic-app-as2-send-receive/azuredeploy.json [hunk 5]
```diff
@@ -0,0 +1,1279 @@
...
+            }
+          },
+          "contentVersion": "1.0.0.0",
+          "outputs": { }
+        },
+        "parameters": {
+          "$connections": {
+            "value": {
+              "as2": {
+                "id": "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Web/locations/', parameters('logicAppLocation'), '/managedApis/', 'as2')]",
```
Comment (gatneil) [inline 201-logic-app-as2-send-receive/azuredeploy.json]: same here

File: 201-logic-app-as2-send-receive/azuredeploy.json [hunk 6]
```diff
@@ -0,0 +1,1279 @@
...
+                "connectionId": "[concat(resourceGroup().id, '/providers/Microsoft.Web/connections/', parameters('fabrikam_AS2_Connection_Name'))]",
```
Comment (gatneil) [inline 201-logic-app-as2-send-receive/azuredeploy.json]: same here

File: 201-logic-app-as2-send-receive/azuredeploy.json [hunk 7]
```diff
@@ -0,0 +1,1279 @@
...
+            }
+          },
+          "contentVersion": "1.0.0.0",
+          "outputs": { }
+        },
+        "parameters": {
+          "$connections": {
+            "value": {
+              "as2": {
+                "id": "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Web/locations/', parameters('logicAppLocation'), '/managedApis/', 'as2')]",
```
Comment (gatneil) [inline 201-logic-app-as2-send-receive/azuredeploy.json]: same here

File: 201-logic-app-as2-send-receive/azuredeploy.json [hunk 8]
```diff
@@ -0,0 +1,1279 @@
...
+                "connectionId": "[concat(resourceGroup().id, '/providers/Microsoft.Web/connections/', parameters('fabrikam_AS2_Connection_Name'))]",
```
Comment (gatneil) [inline 201-logic-app-as2-send-receive/azuredeploy.json]: same here

File: 201-logic-app-as2-send-receive/azuredeploy.json [hunk 9]
```diff
@@ -0,0 +1,1279 @@
...
+      ]
+    },
+    {
+      "type": "MICROSOFT.WEB/CONNECTIONS",
+      "apiVersion": "2016-06-01",
+      "name": "[parameters('contoso_AS2_Connection_Name')]",
+      "location": "[parameters('logicAppLocation')]",
+      "properties": {
+        "api": {
+          "id": "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Web/locations/', parameters('logicAppLocation'), '/managedApis/', 'as2')]"
```
Comment (gatneil) [inline 201-logic-app-as2-send-receive/azuredeploy.json]: same here

File: 201-logic-app-as2-send-receive/azuredeploy.json [hunk 10]
```diff
@@ -0,0 +1,1279 @@
...
+      }
+    },
+    {
+      "type": "MICROSOFT.WEB/CONNECTIONS",
+      "apiVersion": "2016-06-01",
+      "name": "[parameters('fabrikam_AS2_Connection_Name')]",
+      "location": "[parameters('logicAppLocation')]",
+      "properties": {
+        "api": {
+          "id": "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Web/locations/', parameters('logicAppLocation'), '/managedApis/', 'as2')]"
```
Comment (gatneil) [inline 201-logic-app-as2-send-receive/azuredeploy.json]: same here

File: 201-logic-app-as2-send-receive/azuredeploy.json (hunk 3 above)
Comment (vinaysingh123) [inline 201-logic-app-as2-send-receive/azuredeploy.json]: @gatneil [By Design] - These are subscription level provider APIs. Hence the path - subscriptions/subid/providers.

File: 201-logic-app-as2-send-receive/azuredeploy.json (hunk 4 above)
Comment (vinaysingh123) [inline 201-logic-app-as2-send-receive/azuredeploy.json]: @gatneil This is by design.

File: 201-logic-app-as2-send-receive/azuredeploy.json (hunk 5 above)
Comment (vinaysingh123) [inline 201-logic-app-as2-send-receive/azuredeploy.json]: @gatneil This is by design.

File: 201-logic-app-as2-send-receive/azuredeploy.json (hunk 6 above)
Comment (vinaysingh123) [inline 201-logic-app-as2-send-receive/azuredeploy.json]: @gatneil This is by design.

File: 201-logic-app-as2-send-receive/azuredeploy.json (hunk 7 above)
Comment (vinaysingh123) [inline 201-logic-app-as2-send-receive/azuredeploy.json]: @gatneil This is by design.

File: 201-logic-app-as2-send-receive/azuredeploy.json (hunk 8 above)
Comment (vinaysingh123) [inline 201-logic-app-as2-send-receive/azuredeploy.json]: @gatneil This is by design.

File: 201-logic-app-as2-send-receive/azuredeploy.json (hunk 9 above)
Comment (vinaysingh123) [inline 201-logic-app-as2-send-receive/azuredeploy.json]: @gatneil This is by design.

File: 201-logic-app-as2-send-receive/azuredeploy.json (hunk 10 above)
Comment (vinaysingh123) [inline 201-logic-app-as2-send-receive/azuredeploy.json]: @gatneil This is by design.

File: 201-logic-app-as2-send-receive/README.md (hunk 1 above)
Comment (vinaysingh123) [inline 201-logic-app-as2-send-receive/README.md]: @gatneil  Fixed it.

File: 201-logic-app-as2-send-receive/README.md (hunk 2 above)
Comment (vinaysingh123) [inline 201-logic-app-as2-send-receive/README.md]: @gatneil  Fixed.

**Comment by gatneil** [comment]
Looks good to me; merging :)

**Pull request merged**